packer/freebsd.pkr.hcl

```diff
@@ -48,7 +49,7 @@ build {
 
       zone = tag.value.zone
       machine_type = tag.value.machine
-      instance_name = "build-${local.name}-${tag.value.name}"
+      instance_name = "build-${local.name}-${tag.value.name}-${var.image_date}"
```

Comment:
What format does image_date have? It seems this either could lead to duplicates if it really is just the date, or to too long instance names (IIRC they're limited to 63 chars or such)

Reply:
> It is the same DATE variable which is used in image names and it is like 2022-08-01t14-20

Ok. I guess it'd be better to add seconds for this purpose?

> For now, the longest name of the instance is build-pg-ci-sid-newkernel-uring-2022-08-01t14-20 which is 48 chars.

Perhaps we could remove "pg-ci-" from the image name?

Reply:
> > It is the same DATE variable which is used in image names and it is like 2022-08-01t14-20
> 
> Ok. I guess it'd be better to add seconds for this purpose?

Yes, that could work.

> 
> > For now, the longest name of the instance is build-pg-ci-sid-newkernel-uring-2022-08-01t14-20 which is 48 chars.
> 
> Perhaps we could remove "pg-ci-" from the image name?

Are we removing "pg-ci" from image names or instance names?

Reply:
> Are we removing "pg-ci" from image names or instance names?

err, instance, not image names.

Reply:
Done.